lmdeploy/tokenizer.py

```diff
@@ -231,8 +231,14 @@ def _check_transformers_version(self, model_dir: str):
 
         current_transformers_version = version.parse(transformers.__version__)
         cfg = get_model_arch(model_dir)[1]
-        required_transformers_version = version.parse(
-            getattr(cfg, 'transformers_version', 'str'))
+        cfg_ver = getattr(cfg, 'transformers_version', None)
```

Comment:
Hi, @zhyncs 
Would you mind adding unitests for this function?

Reply:
ok

Reply:
The assertion ensures that the tokenizer is not `None`, confirming its successful creation and passing the check.